### PR TITLE
Fix embeddings notebook

### PR DESCRIPTION
**Pull request by pierrepo**
- Clean the environment
- Use functions for embeddings and plots
- Add additional sentences for the first example.

### CODE REVIEW

**Comment by pierrepo** [comment]
@Essmaw this PR is ready to review and merge

File: README.md [hunk 1]
```diff
@@ -41,17 +49,22 @@ python src/parse_clean_markdown.py --in data/markdown_raw --out data/markdown_pr
...
-OPENAI_API_KEY=<your-openai-api-key>
-```
+#### Embeddings :
 
-> Remark: The `.env` file is ignored by git.
+Run the Jupyter notebook `src/analysis_embeddings.ipynb` to visualize embeddings in a 2D and 3D.
 
-### Step 4: Create Chroma DB
+
+#### Chunk size
```
Comment (Essmaw) [inline README.md:60]: To run this notebook successfully, you'll need to have created the various chroma databases of different sizes.

Wouldn't it be better to put "### Create Chroma DB" before?

Comment (pierrepo) [reply README.md:60]: Oh yes. Could you change this after the PR?

**Comment by Essmaw** [comment]
Thanks for your modifications @pierrepo , it's better and clear now !

**Pull request merged**